marlowe-dashboard-client/default.nix

```diff
@@ -24,7 +24,10 @@ let
 
   start-backend = pkgs.writeShellScriptBin "marlowe-pab-server" ''
     echo "marlowe-pab-server: for development use only"
-    $(nix-build ../default.nix --quiet --no-build-output -A marlowe-dashboard.marlowe-invoker)/bin/marlowe-pab --config plutus-pab.yaml all-servers
+    (trap 'kill 0' SIGINT;
+      $(nix-build ../default.nix --quiet --no-build-output -A marlowe-dashboard.marlowe-invoker)/bin/marlowe-pab --config plutus-pab.yaml webserver &
+      $(nix-build ../default.nix -A marlowe-dashboard.marlowe-run-backend-invoker)/bin/marlowe-dashboard-server webserver -c ./config.json
+    )
```

Comment:
Hi @shlevy, I made this as a placeholder to be able to run the PAB + the Marlowe Run BE at the same time, but is not very robust, if one of the process dies you can't kill the other one.

How should we launch these on development? Take in mind that we'll also need WBE and chainindex

Reply:
Will take a look next week.